chrome/content/components/wzQuicktextVar.js

```diff
@@ -68,7 +70,7 @@ function wzQuicktextVar()
   this.mWindow      = null;
 
   // Need the Main Quicktext component
-  this.mQuicktext = Components.classes["@hesslow.se/quicktext/main;1"].getService(Components.interfaces.wzIQuicktext);
+  this.mQuicktext = gQuicktext;
```

Comment:
Can you just go ahead and use gQuicktext everywhere?

Reply:
This will break quicktext scripts (user saved scripts outside of quicktext) as they all use this.mQuicktext to get the global gQuicktext. So I must map that into this.mQuicktext as before.